releases/v20.1.0-beta.1.md

```diff
@@ -0,0 +1,401 @@
+---
+title: What&#39;s New in v20.1.0-beta.1
+toc: true
+summary: Additions and changes in CockroachDB version v20.1.0-beta.1 since version v20.1.0-alpha.20200123
+---
+
+## February 17, 2020
+
+In addition to various updates, enhancements, and bug fixes, this first v20.1 beta release includes the following major highlights:
+
+- **Online primary key changes**: You can now change a table’s primary key using the `ALTER TABLE ... ALTER PRIMARY KEY` statement. Changing a table’s primary key rewrites its primary and some secondary indexes behind-the-scenes and can take a while, but the table remains online with no interruption to data access. For now, this feature is considered experimental and is behind a [cluster setting](../v20.1/cluster-settings.html). To try it out, run `SET experimental_enable_primary_key_changes = true`. The syntax is `ALTER TABLE table_name ALTER PRIMARY KEY USING COLUMNS (x, y)`.
+- **Full cluster restore**: You can now use CockroachDB's [Enterprise `RESTORE`](../v20.1/restore.html) feature to restore a full cluster `BACKUP` to a new cluster, including all configuration and system information such as user privileges, zone configurations, and cluster settings. Restoring a full cluster backup to an existing cluster is not supported.
+- **Encrypted backup files**: You can now use an encryption key to encrypt data in backup files created via CockroachDB's [Enterprise `BACKUP`](../v20.1/backup.html) feature.
```

Comment:
Also mention that you can use the same key you used on a backup to decrypt the backup upon restore